sdks/wasm/src/driver.c

```diff
@@ -674,3 +674,123 @@ mono_wasm_enable_on_demand_gc (void)
 {
 	mono_wasm_enable_gc = 1;
 }
+
+//
+// The following timezone information routines depends on the Mono_WebAssembly_ZoneInfo module being available.
+//
+
+// Returns the local timezone default is UTC.
+EM_JS(int, mono_wasm_timezone_get_local_name, (), {
```

Comment:
`int` or `gsize`? I mean, when wasm is 64bits, and 32bits, what will this be?

Reply:
gsize is not defined.